packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -404,12 +404,15 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _integrateInput: function() {
+        const isValid = this.option('isValid') && this.option('validationStatus') !== 'invalid';
```

Comment:
let's simplify it, since `isValid` and `validationStatus` are synced, there no reason to check both
`const { isValid }  = this.option();`
`this._toggleValidationClasses(!isValid);`

Reply:
[aefd466](https://github.com/DevExpress/DevExtreme/pull/27402/commits/aefd466ba2d7af1e551fc36a6889d74adbd26d65)